opentelemetry-sdk/src/opentelemetry/sdk/metrics/__init__.py

```diff
@@ -44,43 +44,34 @@ def _validate_update(self, value: metrics_api.ValueT) -> bool:
 
 
 class CounterHandle(metrics_api.CounterHandle, BaseHandle):
-    def update(self, value: metrics_api.ValueT) -> None:
+    def add(self, value: metrics_api.ValueT) -> None:
+        """See `opentelemetry.metrics.CounterHandle._add`."""
```

Comment:
```suggestion
        """See `opentelemetry.metrics.CounterHandle.add`."""
```

And the same for the others below. Where did these underscores come from?

Reply:
Was left in by mistake.